packages/fluxible-router/package.json

```diff
@@ -24,10 +27,10 @@
     }
   ],
   "dependencies": {
+    "@babel/runtime": "^7.14.0",
```

Comment:
Should this be a devDependency at the repo root?

Reply:
@babel/runtime is a runtime dependency so it can't be devDependency. And we need to have on each package.json since otherwise it would not be installed when one do `npm install fluxible-router`.

Reply:
whats the reason for needing it to be a runtime dependency?

Reply:
Oh, it's the same as we did for `fluxible-addons-react`. I explained it here: https://github.com/yahoo/fluxible/pull/683#issuecomment-830857300

Reply:
oh right, totally forgot your great explanation. do we need to track the hardcoded babel version to remove that later? any follow ups with this approach we need to keep an eye on later?

Reply:
We could do as react-redux: https://github.com/reduxjs/react-redux/blob/master/.babelrc.js#L16. They read the version from package.json removing the possibility to have mismatched versions between the installed one and the consumed one. 

It's nice, but, in my opinion, giving that we don't update this package too often, we could keep it simple (as we did on addons-react).

Regarding future, it's nice to keep the package version updated since we can benefit more from newer utilities or see them completely vanish when browsers evolve. But the same is valid for all the other babel packages. So I would say that the only thing to keep in mind is, whenever we use update babel deps, we should update this one as well (and remember to update babelrc to point to the right version).

But if prefer, I can prepare another commit that reads the version from package.json as well. What do you say?

Reply:
I sort of like the automatic version update that react-redux does. Its one less thing to remember when upgrading babel, because the version will get stale as new versions come out. I think you could do that in a follow up PR for both packages.